Displays 'No current vacancies' if no job roles listed

DennisTsiang opened the pull request:
If roles is nil then it displays 
- 'No current vacancies' as the title
- "We don't currently have any team openings" underneath.

Commit:
Displays 'No current vacancies' if no job roles listed

neilvanbeinum commented on source/assets/stylesheets/styles/_jobs_page_tile.scss:
```diff
@@ -110,3 +110,10 @@
   }
 }
+
+.jobs-page-tile-no-roles__paragraph {
```
We should stick to the block name in the file for this rule (or make a new file for the new block). How do you feel about `.jobs-page-tile__no-roles-paragraph`?

neilvanbeinum reviewed (changes requested):
Mostly looks good!

I noticed that at mobile widths there's no margin between this section and the next. I think a margin-bottom on the paragraph element for these widths should do the trick.

DennisTsiang commented:
Amended those changes and squashed them to the previous commit

neilvanbeinum commented on source/jobs.erb:
```diff
@@ -22,18 +22,18 @@ unboxed_meeting_image:
+#        We embrace design thinking and collaboration, as part of our
+#        multi-skilled agile teams.
+#      link: /jobs/designer
+#      image: /assets/images/jobs/designer.jpg
```
We'll need to uncomment these :)

DennisTsiang replied:
Ah sorry, I left that in during testing

Commit:
Rename css style names for _jobs_page stylesheets

- replaced filename _jobs_page with _roles
- replaced css classes beginning with jobs_page to roles
- changed style classes in jobs.erb to new names

NeilvB commented:
Thanks for taking the time to do the class rename! I think that's an improvement 👍
